### DISCUSSION TITLE
Zemismart ZMR4 and Zigbee2MQTT

### DISCUSSION THREAD

**Discussion by luks017** [Koenkk/zigbee2mqtt#23753] [Devices]
Hi Everyone,

Need to preface this with I am fairly new to Z2M.

I have purchased a couple Zemismart ZMR4 wireless switches and can’t seem to get them to work. Z2M showing these as compatible but under a different model identifier:  https://www.zigbee2mqtt.io/devices/ZMR4.html

The model of my switches is: _TZ3000_et7afzxz and not ‘_TZ3000_11pg3ima’ as defined in the Herdsman Converters.

Any help would be extremely appreciated.

**Comment by Koenkk** [comment]
Added!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now.

**Comment by vanderfran** [comment]
Running on docker

With this versions :
- Zigbee2MQTT version ->  1.39.1-dev commit: [65b1e60](https://github.com/Koenkk/zigbee2mqtt/commit/65b1e60)
- Coordinator type -> zStack3x0
- Coordinator revision -> 20230507 
- Frontend version -> 0.7.4
- zigbee-herdsman-converters version - >  20.8.0
- zigbee-herdsman version -> 0.57.0

It got recognized as supported but only expose:
- Battery and
- Linkquality

For action I have null a interesting popup why moreover

![image](https://github.com/user-attachments/assets/b85dacba-ee30-47d1-bffa-c9c2565695f1)


![image](https://github.com/user-attachments/assets/a4e1d9f4-c1d7-46dd-87af-273aa98807c9)

![image](https://github.com/user-attachments/assets/7acb1c3e-72a1-4ac8-946f-73be15254d88)
There are no errors or warnings on the logs, but here is a payload sample:
```txt
info 2024-08-30 21:39:57z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/Office Switch', payload '{"action":null,"battery":100,"device":{"applicationVersion":80,"dateCode":"","friendlyName":"Office Switch","hardwareVersion":1,"ieeeAddr":"0x28dba7fffea60434","manufacturerID":4098,"manufacturerName":"_TZ3000_et7afzxz","model":"ZMR4","networkAddress":52497,"powerSource":"Battery","stackVersion":0,"type":"EndDevice","zclVersion":3},"last_seen":"2024-08-30T21:39:57-04:00","linkquality":189}'
```

**Comment by rjhelmke** [comment]
I have 1.40.0 installed and I can add this device, but I am not getting any actions when I press any of the buttons. Should this be working? It does work with Tuya if I add it to a Zigbee Tuya hub.
